tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -394,19 +478,25 @@ def test_attention_outputs(self):
                 list(self_attentions[0].shape[-3:]),
                 [self.model_tester.attention_num_heads, encoder_seq_length, encoder_key_length],
             )
+    
+    @slow
+    def test_model_from_pretrained(self):
+        model = Mamba2Model.from_pretrained("state-spaces/mamba2-130m")
+        self.assertIsNotNone(model)
 
     @unittest.skip(reason="Mamba2 has its own special cache type")
     @parameterized.expand([(1, False), (1, True), (4, False)])
     def test_new_cache_format(self, num_beams, do_sample):
         pass
-
+    
+    # TODO: add test_model_outputs_equivalence test
```

Comment:
Only necessary when we introduce the new output type, but we can leave it there. Maybe a small note to be clear.

Reply:
That's on me, we need it either way to. Jamba skips it and Mamba rewrites due to custom cache.